src/server/api/routers/experiments.router.ts

```diff
@@ -77,6 +79,194 @@ export const experimentsRouter = createTRPCRouter({
     };
   }),
 
+  fork: protectedProcedure.input(z.object({ id: z.string() })).mutation(async ({ input, ctx }) => {
+    // Anyone can fork an experiment
+    requireNothing(ctx);
```

Comment:
We probably should use `requireCanViewExperiment(ctx)` here instead, which is currently the same as `requireNothing(ctx)` but we'll probably let people have totally-private experiments in the future.

Reply:
Okay, updated